### DISCUSSION TITLE
Add option to next/no-img-element to not warn about svg images

### DISCUSSION THREAD

**Discussion by timmywil** [vercel/next.js#28244] [Ideas]
### Describe the feature you'd like to request

I use a lot of SVG images. I'm a big fan of vector graphics that work at any screen size. I recently added the next eslint plugin and noticed a bunch of warnings about not using next/image for my images. I tried it just to see what it might do and it seems be relatively pointless. I don't need a div wrapper and I don't need a bunch of srcsets pointing to exactly the same image.

### Describe the solution you'd like

I'd prefer to have the plugin warn me when I'm not using 'next/image' for png and jpg images, but not warn when the image path ends in `.svg`.  This could either be a value added to the next/no-img-element eslint rule, or really it could just be the default.

### Describe alternatives you've considered

Another option would be to make the Image element a complete noop for SVG elements aside from SVG minification during build time, which can just go in the same path.

**Comment by renansigolo** [comment]
I agree that an exception to .svg files must be added, either as a default or a parameter to the eslint rule, because if my understanding is correct - according to the [Vercel Documentation](https://vercel.com/docs/next.js/image-optimization#limits) the svg images do not benefit from their optimisations anyway.

**Comment by arpitjacob** [comment]
Hey this would be great to have, as I'd like to see this only for Images not SVGs.

**Comment by lirbank** [comment]
Another vote for this!

**Comment by BozidarSiljanoski** [comment]
+1, I vote for this too

**Comment by capekall** [comment]
I just stumbled upon the same thing.
+1 as well

**Comment by cbeardsmore** [comment]
+1
